### PR TITLE
Updated Alert.js with docs on options parameter

### PR DESCRIPTION
**Pull request by cameronmoreau**
While reading the docs on the [Alert API](https://facebook.github.io/react-native/docs/alert.html), I could not find any documentation for the options parameter, and after a few minutes of searching I found it towards the bottom and looking at the code.

I added the options `cancelable` documentation to the method documentation.

### CODE REVIEW

File: Libraries/Alert/Alert.js [hunk 1]
```diff
@@ -72,7 +72,17 @@ type Options = {
  * ```
  */
 class Alert {
-
+
```
Comment (hramos) [inline Libraries/Alert/Alert.js]: Can you address this?

Comment (cameronmoreau) [inline Libraries/Alert/Alert.js]: @hramos Fixed!

File: Libraries/Alert/Alert.js [hunk 2]
```diff
@@ -72,7 +72,16 @@ type Options = {
...
-
+  /**
+   * Displays an alert dialog
+   *
+   * Options is an object that accepts `cancelable`. If true,
+   * the modal will not close by pressing the background, only
+   * by pressing a button.
+   *
+   * Example:
+   * `{cancelable: true}`
```
Comment (mkonicek) [inline Libraries/Alert/Alert.js]: From the docs above (line 55) it looks like this is actually the opposite? When `false`, the modal can't be closed by tapping outside.

**Comment by mkonicek** [comment]
Good catch, thanks for improving the docs!

It looks like someone fixed this on master in the meantime, see the `cancelable` option mentioned here: http://facebook.github.io/react-native/releases/next/docs/alert.html

I'll close this since the description we have on master now looks good:

> Note that by default alerts on Android can be dismissed by clicking outside of their alert box. To prevent this behavior, you can provide an optional options parameter { cancelable: false } to the Alert method.

**Pull request closed without merging**